warehouse/manage/views/organizations.py

```diff
@@ -573,7 +574,16 @@ def create_or_manage_subscription(self):
         if not self.request.organization_access:
             raise HTTPNotFound()
 
-        if not self.organization.subscriptions:
+        live_subscriptions = (
+            self.request.db.query(StripeSubscription)
+            .filter(StripeSubscription.organization == self.organization)
+            .filter(
+                StripeSubscription.status.not_in([StripeSubscriptionStatus.Canceled])
+            )
+            .all()
+        )
+
+        if not live_subscriptions:
```

Comment:
Should we use `organization.active_subscription` here instead, or create a similar property?

Reply:
active is not quite "manageable" so I implemented "manageable_subscriptions" in 1e8b28432f343392ba6bba19b25e455533921e3a